components/OrgHeader.tsx

```diff
@@ -1,23 +1,37 @@
 import React, { FC } from 'react';
-import DirectoryOrgTags from './DirectoryOrgTags';
+import Club from '../models/club';
+import Image from 'next/image';
 
-interface Props {
-  name: string;
-  tags: string[];
-}
-
-const DirectoryOrgHeader: FC<Props> = ({ name, tags }) => {
+const DirectoryOrgHeader: FC<{ club: Club }> = ({ club }) => {
   return (
-    <>
-      <div className="md:col-span-4">
-        <h1 className="text-2xl md:text-5xl font-bold">{name}</h1>
-      </div>
-      <div className="md:col-start-[-1] p-3 flex">
-        {tags.map((tag, i) => (
-          <DirectoryOrgTags key={i} tag={tag} />
+    <div className="relative m-5">
+      <Image
+        src="/banner.png"
+        alt="Picture of the club"
+        width={1920}
+        height={490}
+        className="rounded-lg object-cover"
+        priority
+      />
+      <div className="absolute top-0 left-0 flex translate-y-3">
```

Comment:
Would it make sense to refactor these tags into their own component?

Reply:
The tag has to be attached to the image so I want to leave it as unambiguous as possible for future devs